### ISSUE TITLE
Output execution result to variable in code

### ISSUE THREAD

**Issue by maxpavlov** [fluent-ffmpeg/node-fluent-ffmpeg#441]
Hello. I need to save the list of direct show devices in Windows. 

Via command line I do the following:
ffmpeg -list_devices true -f dshow -i dummy -hide_banner

How do I convert it into fluent-ffmpeg and redirect command output to a variable as string to analize then? 
I don't seem to be able to figure out which type of option (input or output) are -list_devices and -f dshow and -hide_banner.

Thanks in advance.

**Comment by eisenmaxx** [comment]
i guess you could use https://www.npmjs.com/package/intercept-stdout for this, or

**Comment by njoyard** [comment]
Yep, as @eisenmaxx hinted this is not supported for now. You may use some lib to run ffmpeg yourself and capture its output (or do it manually). Or feel free to submit a pull request with this as a new feature :p

**Comment by odigity** [comment]
I found this issue while looking for a solution to the same problem -- I really want to supply the -hide_banner option to eliminate a lot of useless noise in the output when capturing errors, and am disappointed to find no straight-forward way to do so.

Couldn't this be as simple as adding an options() method for general custom options not specific to inputs and outputs?
